uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ForgeProjectsTest.java

```diff
@@ -0,0 +1,184 @@
+	public static final String RESPONSE_ARCHETYPE_CATALOG = " Archetype catalog version";
+	public static final String RESPONSE_PROJECT_TYPE2 = "? Project type ";
+	public static final String RESPONSE_ARCHETYPE = "* Archetype:";
+	public static final String RESPONSE_ERROR = "***ERROR***";
+	public static final String RESPONSE_PROJECT_CREATED = "***SUCCESS*** Project named '" + PROJECT_NAME
+			+ "' has been created.";
+
+	public static final String CMD_RETURN = "\n";
+	public static final String CMD_NEW_PROJECT = "project-new \n";
+
+	private ForgeOption project;
+
+	@Rule
+	public ErrorCollector collector = new ErrorCollector();
+
+	@Parameters(name = "{0}")
+	public static Collection<ForgeOption> setupData() {
+		ForgeConsoleView view = new ForgeConsoleView();
+		view.open();
+		view.start();
+		List<ForgeOption> params = getAvailableArchetypes();
+		view.setConsoleText(CMD_RETURN);
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_ERROR));
+		return params;
+	}
+
+	public ForgeProjectsTest(ForgeOption project) {
+		this.project = project;
+	}
+
+	@Before
+	public void setupDeleteProjects() {
+		new ProjectExplorer().deleteAllProjects();
+	}
+
+	private static List<ForgeOption> getAvailableArchetypes() {
+		ForgeConsoleView view = new ForgeConsoleView();
+		view.setConsoleText(CMD_NEW_PROJECT);
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_PROJECT_NAME));
+		view.setConsoleText(PROJECT_NAME + CMD_RETURN);
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_TOP_PACKAGE));
+		view.setConsoleText(CMD_RETURN);
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_VERSION));
+		view.setConsoleText(CMD_RETURN);
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_FINAL_NAME));
+		view.setConsoleText(CMD_RETURN);
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_LOCATION));
+		view.setConsoleText(CMD_RETURN);
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_TARGET_ROOT));
+		view.setConsoleText(CMD_RETURN);
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_PROJECT_TYPE));
+		AbstractWait.sleep(TimePeriod.SHORT);
+		for (ForgeOption option : view.getOptions()) {
+			if (CHOICE_PROJECT_TYPE.equals(option.getName())) {
+				view.setConsoleText(option.getNumber() + CMD_RETURN);
+				break;
+			}
+		}
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_BUILD_SYSTEM));
+		view.setConsoleText(CMD_RETURN);
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_ARCHETYPE_CATALOG));
+		view.setConsoleText(CMD_RETURN);
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_PROJECT_TYPE2));
+		for (ForgeOption option : view.getOptions()) {
+			if (RUNTIME_TYPE.equals(option.getName())) {
+				view.setConsoleText(option.getNumber() + CMD_RETURN);
+			}
+		}
+		AbstractWait.sleep(TimePeriod.SHORT);
+		new WaitUntil(new ForgeConsoleHasText(RESPONSE_ARCHETYPE));
+		return view.getOptions();
+	}
+
+	@Test
+	public void testArchetype() {
+		ForgeConsoleView view = new ForgeConsoleView();
+		view.open();
+		createProject(project);
+		if (LogChecker.noFuseError()) {
```

Comment:
LogChecker.noFuseErrors() returns "true" if there aren't "fuse" errors in error log, maybe did you expecting opposite here ?